inst/rmd/scratch_gimap_annotation_review.Rmd

```diff
@@ -0,0 +1,304 @@
+---
+title: "gimap_annotation_review"
+output: html_document
+date: "`r Sys.Date()`"
+---
+
+```{r}
+
+library(gimap)
+library(tidyverse)
+
+```
+
+## Load in the GI Mapping annotation table
+
+```{r}
+annot <- readRDS("~/Downloads/results/pgRNA_annotations/tables/rds/d.HeLa_annot")
+
+```
+
+## Get the gimap annotation results
+
+### Get the gimap data setup
+
+```{r}
+example_data <- get_example_data("count")
+
+example_counts <- example_data %>%
+  select(c("Day00_RepA", "Day05_RepA", "Day22_RepA",
+           "Day22_RepB", "Day22_RepC")) %>%
+  as.matrix()
+
+example_pg_id <- example_data %>%
+  dplyr::select("id")
+  example_pg_metadata <- example_data %>%
+  select(c("id", "seq_1", "seq_2"))
+
+example_sample_metadata <- data.frame(
+  id = 1:5,
+  day = as.factor(c("Day00", "Day05", "Day22", "Day22",
+                    "Day22")),
+  rep = as.factor(c("RepA", "RepA", "RepA", "RepB", "RepC"))
+  )
+
+example_pg_metadata <- get_example_data("meta")
+
+gimap_dataset <- setup_data(counts = example_counts,
+                            pg_ids = example_pg_id,
+                            pg_metadata = example_pg_metadata,
+                            sample_metadata =
+                              example_sample_metadata)
+
+```
+
+### Define the annotation function from the code
+
+Copy/pasted from `03-annotate.R` from the `cansavvy/annotation-schema` branch
```

Comment:
If you don't want to copy and paste code, you could build this branch off of that one and then `devtools::load_all()`

Reply:
Oh I see you did create this branch off that one. So you can just load all so you don't have to copy and paste. This would make it easier to keep track of updates to the code because if you pull the branch updates then no need to copy and paste again.